build.gradle.kts

```diff
@@ -51,6 +52,7 @@ dependencies {
     testImplementation("commons-codec:commons-codec:1.+")
     testImplementation("org.apache.commons:commons-lang3:3.+")
```

Comment:
For dependencies pulled up to `implementation` we don't need to declare them again for tests.
```suggestion
```